apps/nextjs-website/react-components/components/Header/Header.helpers.tsx

```diff
@@ -0,0 +1,33 @@
+import { Stack, useTheme, type Theme } from '@mui/material';
+import { DialogBubbleProps } from '../../types/Header/Header.types';
+
+const useStyles = (mui: Theme) => ({
+  bubbleContainer: {
+    transform: 'rotate(180deg)',
+    position: 'absolute',
+    marginTop: '42px',
+    padding: mui.spacing(2),
+    direction: 'rtl',
+    textAlign: 'left',
+    borderRadius: '4px',
+  },
+  bubble: { transform: 'rotate(180deg)' },
+});
+
+export const DialogBubble = ({
+  children,
+  ...stackProps
+}: DialogBubbleProps) => {
+  const mui = useTheme();
```

Comment:
```suggestion
  const muiTheme = useTheme();
```
For consistency with other similar functions